test/controllers/EditSetControllerSpec.scala

```diff
@@ -948,6 +948,20 @@ class EditSetControllerSpec extends BaseSpec {
               ExpectedSelectOption("2", "British Museum, Department of Libraries and Archives"),
               ExpectedSelectOption("3", "British Library, National Sound Archive")
             ),
+            separatedMaterial = Seq(
+              ExpectedSeparatedMaterial(
+                linkHref = Some("#;"),
+                linkText = Some("COAL 80/80/5")
+              ),
+              ExpectedSeparatedMaterial(
+                linkHref = Some("#;"),
+                linkText = Some("COAL 80/80/6")
+              ),
+              ExpectedSeparatedMaterial(
+                linkHref = Some("#;"),
+                linkText = Some("COAL 80/80/7")
+              )
+            ),
```

Comment:
`when all data is valid` scenario to check for existence of Related and Separated. Suggestion to include test when only Separated or Related exists